socorro/external/rabbitmq/crashstorage.py

```diff
@@ -55,12 +40,6 @@ class RabbitMQCrashStorage(CrashStorageBase):
 
     def __init__(self, config, namespace='', quit_check_callback=None):
         super().__init__(config, namespace=namespace, quit_check_callback=quit_check_callback)
-
-        # Note: this may continue to grow if we aren't acking certain UUIDs.
-        # We should find a way to time out UUIDs after a certain time.
-        self.acknowledgement_token_cache = {}
-        self.acknowledgment_queue = Queue()
```

Comment:
Consuming the queue is no longer in the domain of the crashstore, so we don't need to deal with acks.